engine/engine/content/builtins/manifests/web/engine_template.html

```diff
@@ -94,144 +94,13 @@ <h1>WebGL not supported ⚠️</h1>
 	</div>
 	<!-- -->
 	<script id='engine-loader' type='text/javascript' src="dmloader.js"></script>
-	<!-- -->
```

Comment:
it seems like some changes should be made in the documentation (after PR will be merged): https://defold.com/manuals/html5/#extra-parameters

https://github.com/defold/doc/blob/master/docs/en/manuals/html5.md

Reply:
https://github.com/defold/defold/pull/8682